### DISCUSSION TITLE
Audio engine losing connection after extended none play

### DISCUSSION THREAD

**Discussion by mattrixed** [GrandOrgue/grandorgue#1194] [Help]
Win10 21H2
Focusrite 18i20

On latest 3.6.2 Last couple versions have been noticing that leaving pc on and everything working a-ok - leave it for a few hours and come back to it everything on and working but NO audio output.  Even the audio bars acknowledge output but no physical audio from audio interface - did it this morning after 1/2 hour seems a little random and just now again after 4-5hours not being played

win10 latest up to date
Power profile high power
No sleep full on and running - not had this issue before 3.6
panic doesn’t fix and sometimes crashes go
Restarting GO doesn’t fix issue
Launchpad still works aka another usb device

Shut down GO turn off audio interface then reverse start up again and then works…

**Comment by oleg68** [comment]
> Restarting GO doesn’t fix issue

Please try to run any another audio applications with the same audio device. Do they sound?

**Comment by mattrixed** [comment]
Good idea Oleg Will do next time it stops. Hasn’t yet after an hour or so….

**Comment by oleg68** [comment]
@mattrixed Have you managed to test with another applications?

**Comment by mattrixed** [comment]
As is always the case, it hasn’t done it since…. But I can definitely play audio through the focusrite from a browser and YouTube for eg….   Will start up the organ pc and try again… but maybe it was in win10 side….?  It definitely did it though multiple times….!!! No other change….
